### PR TITLE
Add container to create Notifications and Alarm Definitions

### PR DESCRIPTION
**Pull request by craigbr**
Container runs once to create Notifications and Alarm Definitions
in Monasca. The creation command checks first if the Notification
or Alarm Definition exists and matches what is expected. If it
matches, nothing is done so this container is idempotent.

### CODE REVIEW

File: monasca-alarms/definitions.yml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    expression: "http_status > 0"
+    match_by:
+      - "service"
+      - "component"
+      - "hostname"
+      - "url"
+    alarm_actions:
+      - root email
+    ok_actions:
+      -  root email
```
Comment (kaiyan-sheng) [inline monasca-alarms/definitions.yml]: extra space here and the rest of the ok_actions lines.

File: monasca-alarms/monasca_alarm_definition.py [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+            - Whether the alarm definition should exist.  When absent, removes the alarm definition. The name
+              is used to determine the alarm definition to remove
+    undetermined_actions:
+        required: false
+        description:
+            -  Array of notification names that are invoked for the transition to the UNDETERMINED state
+Examle yaml file:
+
+notifications:
+  - name: Notify via Email
```
Comment (timothyb89) [inline monasca-alarms/monasca_alarm_definition.py]: :smile:

File: monasca-alarms/definitions.yml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+# under the License.
+#
+
+#
+# Default Notifications and Alarms
+#
+notifications:
+  - name: root email
+    type: email
+    address: root@localhost
```
Comment (timothyb89) [inline monasca-alarms/definitions.yml]: My only real concern here, will there be issues if notifications can't be sent? `root@localhost` will never be valid, so I'd like to be  sure it won't cause trouble if the defaults are used.

**Comment by craigbr** [comment]
The notification engine will try once and if it can't send the email, it will just log a message and drop the notification

**Pull request merged**